lib/public/views/Runs/ActiveColumns/runsActiveColumns.js

```diff
@@ -508,7 +544,14 @@ export const runsActiveColumns = {
         classes: 'w-15 f6',
         visible: false,
         profiles: [profiles.none, 'lhcFill', 'environment'],
-        filter: odcTopologyFullName,
+
+        /**
+         * ODC topology full name filter component
+         *
+         * @param {RunsOverviewModel} runsOverviewModel the runs overview model
+         * @return {Component} the filter component
+         */
+        filter: (runsOverviewModel) => rawTextFilter(runsOverviewModel.filteringModel.get('odcTopologyFullName')),
```

Comment:
I believe you are missing the `w-100` property as currently the filter is not displayed in a similar fashion to the other raw text filters

Reply:
Indeed, thanks a lot for notice! Both fixed